.python-version

```diff
@@ -1 +1 @@
-3.6.3
+3.7.2
```

Comment:
Any reason not to upgrade to 3.8.0?

Reply:
Only that we're currently using this version on runner. Using 3.8 would save the travis install step.